vite.config.development.js

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'vite'
+import { config as prodConfig } from './vite.config'
+
+// https://vitejs.dev/config/
+export const config = ({ mode }) => {
+  const config = prodConfig({ mode })
+  //
+  config.resolve.alias = {
+    ...config.resolve.alias,
+    ...{
+      '/src/services/index.ts': '/src/services/development.ts',
+    },
```

Comment:
**Suggestion**: No need for destructuring an object literal when you can just use its fields:

```suggestion
    '/src/services/index.ts': '/src/services/development.ts',
```

Reply:
I used `mergeConfig` for this instead.